recipes/nextcloud-deck/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "id": "nextcloud-deck",
+  "name": "Nextcloud Deck",
+  "version": "1.0.0",
+  "license": "MIT",
+  "repository": "https://github.com/mindcombatant/ferdium-recipes/tree/master/recipes/nextcloud-deck/",
+  "config": {
+    "hasNotificationSound": true,
+    "hasTeamId": true,
+    "urlInputPrefix": "https://",
+    "urlInputSuffix": "/apps/news/",
```

Comment:
The `serviceURL` below uses `/apps/deck`, but the user will see `/apps/news` as can be seen here:
![Capture d’écran 2023-05-15 à 00 21 06](https://github.com/ferdium/ferdium-recipes/assets/34252790/4e619d35-7713-43e9-8769-21edff6d07c8)
Please make this consistent.

Reply:
oh, whoops, i could've sworn i got all of them, thanks for pointing this out! should be fixed now.